src/calculateRentalCost.js

```diff
@@ -27,8 +27,35 @@
  * @return {number}
  */
 
+const RENT_COST = 40;
+const FIRST_OFF = 20;
+const SECOND_OFF = 50;
+const LONG_TERM_3 = 3;
+const LONG_TERM_6 = 6;
+const LONG_TERM_7 = 7;
+
+
+
 function calculateRentalCost(days) {
-  // write code here
+  let finalCost;
```

Comment:
```suggestion
  let finalCost = days * RENT_COST;
```
you may calculate the base cost initially instead of doing it in every block